openshift/scripts/oc_cleanup.sh

```diff
@@ -28,40 +28,20 @@ if [ "${APPLY}" ]; then
 else
 	DELETE_OR_GET="get"
 fi
-OC_CLEAN_DEPLOY="oc -n ${PROJ_TARGET} ${DELETE_OR_GET} all,cm -o name -l app=${NAME_OBJ}"
-OC_CLEAN_HOURLY_CRONJOB="oc -n ${PROJ_TARGET} ${DELETE_OR_GET} --ignore-not-found=true cronjob/bcfw-p1-hourly-actuals-${NAME_APP}-${SUFFIX}"
-OC_CLEAN_FORECAST_CRONJOB="oc -n ${PROJ_TARGET} ${DELETE_OR_GET} --ignore-not-found=true cronjob/bcfw-p1-forecasts-${NAME_APP}-${SUFFIX}"
-OC_CLEAN_EC_GDPS_CRONJOB="oc -n ${PROJ_TARGET} ${DELETE_OR_GET} --ignore-not-found=true cronjob/env-canada-gdps-${NAME_APP}-${SUFFIX}"
-OC_CLEAN_EC_HRDPS_CRONJOB="oc -n ${PROJ_TARGET} ${DELETE_OR_GET} --ignore-not-found=true cronjob/env-canada-hrdps-${NAME_APP}-${SUFFIX}"
-OC_CLEAN_EC_RDPS_CRONJOB="oc -n ${PROJ_TARGET} ${DELETE_OR_GET} --ignore-not-found=true cronjob/env-canada-rdps-${NAME_APP}-${SUFFIX}"
+OC_CLEAN_DEPLOY="oc -n ${PROJ_TARGET} ${DELETE_OR_GET} all,cm,pvc -o name -l app=${NAME_OBJ}"
```

Comment:
don't we still need the `--ignore-not-found` flag?

Reply:
I think --ignore-not-found is only an issue on cronjobs.